backend/settings/base.py

```diff
@@ -140,3 +142,10 @@
         'rest_framework_simplejwt.authentication.JWTAuthentication',
     ],
 }
+
+SIMPLE_JWT = {
+    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=60),
+    'REFRESH_TOKEN_LIFETIME': timedelta(minutes=5),
```

Comment:
I think the refresh token should have a longer life span than the access token, especially since the access token is dependent on the refresh token. [Use as a guide](https://django-rest-framework-simplejwt.readthedocs.io/en/latest/settings.html)